frontend/src/components/annotation/annotation_core.vue

```diff
@@ -6781,19 +6807,37 @@
           result = this.initialize_instance(result);
           return result
         },
+        copy_all_instances: function(){
+          let new_instance_list = []
+          for(const instance of this.instance_list){
+            if(instance.soft_delete){
+              continue
+            }
+            let instance_clipboard = this.duplicate_instance(instance);
+            instance_clipboard.selected = false;
+            instance_clipboard.original_frame_number = this.current_frame;
+            new_instance_list.push(instance_clipboard)
+
+          }
+          this.set_clipboard(new_instance_list);
+          this.show_snackbar('All files copied into clipboard.')
```

Comment:
maybe say `instances` not `files` here?

Reply:
Thanks Anthony! Fixed here: https://github.com/diffgram/diffgram/pull/243/commits/612d7b1c577605558ea735c3c943b7dd8d36721a